.infrastructure/cronjob.yml

```diff
@@ -0,0 +1,30 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: todoapp-cronjob
+  namespace: todoapp
+spec:
+  schedule: "*/4 * * * *"
+  concurrencyPolicy: Allow
+  successfulJobsHistoryLimit: 10
+  failedJobsHistoryLimit: 5
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: curl-container
+            image: losiev/busyboxplus:curl
+            command:
+            - /bin/sh
+            - -c
+            - "curl http://todoapp-service.todoapp.svc.cluster.local/api/health"
+            resources:
+              requests:
+                memory: "64Mi"
+                cpu: "250m"
+              limits:
+                memory: "128Mi"
+                cpu: "500m"
+          restartPolicy: OnFailure     
+                  
```

Comment:
For EoF to be displayed correctly, the last line should be blank. Remove space
![image](https://github.com/user-attachments/assets/cd5a8b2e-c6e5-42f6-8e3e-bba9e7db2c5b)